### ISSUE TITLE
Babylon protobuf js automation

### ISSUE THREAD

**Issue by jrwbabylonlab** [babylonlabs-io/simple-staking#236]
Once #230 and #231 is done. We can think about how to perform the upgrade of the protobuf in ts/js autmatically based on newer Babylon version.
Some thoughts:
- Whenever babylon merged a new PR in release version, we will automatically run a set of upgrade and build script
- The script will generate a checksum based on all protobuf files and check if it's different to the previous checksum in the file.
- If different, then bot will raise a PR automatically with the new changes.
